README.md

```diff
@@ -3,39 +3,65 @@
 ## Installation
 1. Clone repository into workspace [It doesn't have to be called moveit_simple_ws]
 ```
-mkdir moveit_simple_ws && cd moveit_simple_ws
-mkdir src && cd src
-catkin_init_workspace
+mkdir -p moveit_simple_ws/src && cd moveit_simple_ws/src
 git clone -b kinetic-devel https://github.com/plusone-robotics/moveit_simple.git
 ```
 
-2. Source Dependencies
+2. Get Source Dependencies
 ```
+cd moveit_simple_ws/src
 wstool init . moveit_simple/.travis.rosinstall
-cd ..
 ```
 
-3. Package Dependencies
+3. Install Package Dependencies
 ```
+cd moveit_simple_ws
 rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
 ```
 
 4. Build
 ```
-catkin build --cmake-args -DCMAKE_BUILD_TYPE=Debug
-or
 catkin build --cmake-args -DCMAKE_BUILD_TYPE=Release
+```
+**NOTE:** To use gdb with this project set `-DCMAKE_BUILD_TYPE=Debug` instead.
 
+5. Source the Workspace
+```
+cd moveit_simple_ws
 source devel/setup.bash
 ```
 
-## Running Tests with Rviz Visualization
+**NOTE:** You will need to re-run step #5 in any new terminal
+
+
+## Building and Running Tests
+
+1.  Run the MoveIt Simple Tests
+```
+roscd moveit_simple
+catkin run_tests --no-deps --this -i
```

Comment:
Can also be run from an arbitrary location in the workspace with `catkin build moveit_simple --no-deps -i --catkin-make-args run_tests`. Would eliminate the [roscd](https://github.com/plusone-robotics/moveit_simple/pull/84/files#diff-04c6e90faac2675aa89e2176d2eec7d8R41) above and [cd](https://github.com/plusone-robotics/moveit_simple/pull/84/files#diff-04c6e90faac2675aa89e2176d2eec7d8R47) below.

Reply:
You are completely correct, these steps border on overly explicit. My personal preference is to write README's in a way where a user can copy-paste bash commands into their terminal from almost any state and it will run as expected. I made some changes that I think you will be happy with